src/shared/cryptsetup-fido2.c

```diff
@@ -129,12 +131,7 @@ int acquire_fido2_key(
 
 int acquire_fido2_key_auto(
                 struct crypt_device *cd,
-                const char *name,
-                const char *friendly_name,
```

Comment:
Please do not drop `friendly_name` here. We really should include that in the PIN prompt. The fact that we currently do not is an omission we should fix.

i.e. instead of dropping this parameter please change the ask_password_auto() line above to include it in the question, so that it is clear for which volume we need the token.

Reply:
and the `name` should not be dropped either, but passed in the `id` parameter of ask_password_auto() (i.e. the third one, which is currently NULL), so that agents implementing the passwword question protocol can identify what we are asking for.

Reply:
What name do you think we should pass in from cryptenroll? Or maybe just make it NULL in that case only?

Reply:
arg_node should be suitable.

Reply:
It seems that none of the other cryptsetup modes included the volume name / friendly name in their calls to ask_password_auto() either. I'll just leave this change out of this PR and potentially fix it in a future one.